src/components/Thumbnail/tests/Thumbnail.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {mountWithApp} from 'test-utilities/react-testing';
+import {NoteMajor} from '@shopify/polaris-icons';
+
+import {Thumbnail} from '../Thumbnail';
+
+describe('<Thumbnail />', () => {
+  describe('source', () => {
```

Comment:
@alex-page / @kyledurand None of these tests have assertions and are therefor causing `type-check` to fail 😢  Does Travis not prevent merging when in-progress/failed?